sql03rdOct/sql03rdOct.txt

```diff
@@ -0,0 +1,117 @@
+//uber tables database
+CREATE DATABASE uber;
+
+USE uber;
+
+CREATE TABLE riders (
+	RiderID INT PRIMARY KEY
+	,RiderName VARCHAR(30)
+	,RiderAge INT
+	,RiderContact INT
```

Comment:
what type of contact is this ?

Reply:
the contact number of the customer to be used in case to contact for queries regarding the pickup or drop just after the booking the trip